flexibleengine/resource_flexibleengine_obs_bucket_notification.go

```diff
@@ -0,0 +1,191 @@
+package flexibleengine
+
+import (
+	"context"
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/fmtp"
+	"log"
+	"strings"
+
+	"github.com/chnsz/golangsdk/openstack/obs"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
```

Comment:
suggest to rearrange the order the imported packages

Reply:
OK, it has been fixed.